plutus-tx-plugin/src/Language/PlutusTx/Plugin.hs

```diff
@@ -336,3 +308,85 @@ runCompiler opts expr = do
 
     let uplcP = UPLC.eraseProgram plcP
     pure (pirP, uplcP)
+
+
+-- Helpers
```

Comment:
I'm not a huge fan of the "helpers" section. Usually you find functions by searching for their names if you're not sure where they are, and if anything you want them close to the place they're used.